### PR TITLE
Updates to deployment policy

### PR DESCRIPTION
**Pull request by gunsch**
Proposing the following updates to the deployment policy, based on Friday's incident:

* Removes some outdated / non-policy bits near the end
* Adds introductory stanza about our overall stance
* Adds section about resolving production bugs, prioritizing reverts, not rolling back

### CODE REVIEW

File: platform/working-with-vsp/policies-work-norms/deployment-policy.md [hunk 1]
```diff
@@ -58,16 +66,12 @@ When in doubt on whether an issue is _critical_ enough for out-of-band deploymen
 1. Once approved, contact the [VSP DevOps oncall](https://dsva.pagerduty.com/schedules#PGIEA8Q) through the [#oncall channel](https://dsva.slack.com/archives/C30LCU8S3) to coordinate the release.
 1. Within two business days of this incident, send a PR with a [postmortem](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/tree/master/Postmortems), including any relevant stakeholders, VSP DEPO leadership, and any VSP team members involved in resolving the incident as reviewers.
 
-## Step-by-step process for deployment
-
-1. To deploy this change to production, the release job is run with the git SHA for the commit to deploy.
-1. The auto-deploy boolean flag must be set to `FALSE` on the job.
-1. The release tag is captured and then used as input for the matching deploy job.
-1. There are "special" jobs for `vets-website` and `vets-api` which will run both the release and deploy of `master` in one operation.
+## Resolving production bugs
```
Comment (gunsch) [inline platform/working-with-vsp/policies-work-norms/deployment-policy.md:69]: curious for y'all's thoughts on location on this section. I'm pretty sure this is absolutely the *wrong* place for this information, but I don't have a better place easily handy. (_possibly_ the Incident Response playbook, but that might be a little out of the way for this information? I'm open to ideas.)

Comment (rianfowler) [inline platform/working-with-vsp/policies-work-norms/deployment-policy.md:69]:
```suggestion
## Resolving post deploy issues 
```
This is tricky. We want to resolve issues as quickly as possible but I don't think reverting is the right call for lower priority production bugs.

Comment (rianfowler) [reply platform/working-with-vsp/policies-work-norms/deployment-policy.md:69]: This seems fine to me for now.

**Review by rianfowler — APPROVED** [review]

File: platform/working-with-vsp/policies-work-norms/deployment-policy.md (hunk 1 above)
Comment (dginther) [reply platform/working-with-vsp/policies-work-norms/deployment-policy.md:69]: I would say this could go in the Engineering SOPs

Comment (gunsch) [reply platform/working-with-vsp/policies-work-norms/deployment-policy.md]: Hmm, I'd like these policies to remain public if possible, and `devops` isn't.

I've tried moving this to a new location, under a doc called [resolving critical issues](https://github.com/department-of-veterans-affairs/va.gov-team/blob/gunsch-deploy-policy/platform/engineering/resolving-critical-issues.md), adding some context, and linking between this doc and that one:

that way, this doc can remain focused on _deployments_, and the other doc can talk about how we fix issues or not. lemme know what you think.

Comment (gunsch) [reply platform/working-with-vsp/policies-work-norms/deployment-policy.md]: I ended up changing this heading entirely after separating it to the "resolving critical issues" doc. it's not really about "production" or "post deploy" anymore.

If we want the guidance to be "keep master deployable", to me that's equivalent to "keep master free of critical issues" (where "critical issues" is defined in this doc already). so I think the guidelines on how to fix (fix in five minutes, or revert) apply whether or not the issue has reached production.

**Review by philipwritescode — APPROVED** [review]

**Pull request merged**